### DISCUSSION TITLE
나무위키 리버티 스킨 사용시 본문이 사라집니다

### DISCUSSION THREAD

**Discussion by hui1601** [List-KR/community#71] [Q&A]
리버트 스킨 활성화시 [마이페이지](https://namu.wiki/member/mypage)의 본문이 다음 규칙에 의해 차단되었다고 나옵니다.
`namu.wiki##div:not([class*=" "]) ~ div div + div[class]:has(> div[id][class]):not(:has(div[class*=" "] > span))`
확인부탁드립니다.

**Comment by piquark6046** [comment]
https://github.com/List-KR/List-KR/issues/954
